index.d.ts

```diff
@@ -256,4 +256,35 @@ declare module 'gqutils' {
 		parseGraphqlSchemas(): schemaMap;
 		parseGraphqlSchema(schema: string): GraphQLSchema;
 	}
+
+	interface schemaConfigInput extends commonOptions {
+		validateGrqphql?: boolean;
+		cache?: any;
+		formatError?: (error: any) => any;
```

Comment:
Add comment that by default it uses gqutils formatError.
And error type would be `Error`